Services/Transaction/Handler/TransactionPatchHandler.php

```diff
@@ -10,8 +10,11 @@
 
```

Comment:
This change is for supporting 404 status codes within PATCH requests. It uses the registry rather than the entity manager because this class never used the entity manager to begin with. Writing this feature with an EM to later be replaced with the registry would have been pointless.